content/docs/hooks-overview.md

```diff
@@ -1,24 +1,23 @@
 ---
 id: hooks-overview
-title: Hooks at a Glance
+title: एक नज़र में Hooks
 permalink: docs/hooks-overview.html
 next: hooks-state.html
 prev: hooks-intro.html
 ---
+React 16.8 में *Hooks* एक नया अतिरिक्त है। वे आपको class लिखने के बिना state और अन्य React प्रतिक्रिया सुविधाओं का उपयोग करने देते हैं।
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+Hooks [बैकवर्ड-कम्पेटिबल](/docs/hooks-intro.html#no-breaking-changes) है।. यह पृष्ठ अनुभवी React उपयोगकर्ताओं के लिए Hooks का अवलोकन प्रदान करता है। यह एक तेज अवलोकन है। यदि आप भ्रमित हो जाते हैं, तो इस तरह पीले बॉक्स की तलाश करे:
 
-Hooks are [backwards-compatible](/docs/hooks-intro.html#no-breaking-changes). This page provides an overview of Hooks for experienced React users. This is a fast-paced overview. If you get confused, look for a yellow box like this:
-
->Detailed Explanation
+>विस्तृत विवरण
 >
->Read the [Motivation](/docs/hooks-intro.html#motivation) to learn why we're introducing Hooks to React.
+>यह जानने के लिए [प्रेरणा](/docs/hooks-intro.html#motivation) पढ़ें कि हम Hooks का React में परिचय क्यों कर रहे हैं।
```

Comment:
`प्रेरणा` => `मोटिवेशन के बारे में`

how about this? I feel `प्रेरणा` is changing the meaning of sentence

Reply:
@arshadkazmi42 
Yeah, makes sense, however, I feel only `[मोटिवेशन]` should be enough to write in the box, as it's not a sentence/statement and just a word representation only.